cities/brühl.json

```diff
@@ -0,0 +1,63 @@
+{
+    "features": [
+        {
+            "geometry": {
+                "coordinates": [
+                    6.89742,
+                    50.82805
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Brühler Wochenmarkt",
+                "location": "Balthasar-Neumann-Platz",
+                "opening_hours": "Tu, Th, Sa 07:00-13:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    6.90468,
+                    50.82831
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Frischemarkt Innenstadt",
+                "location": "Uhlstraße",
+                "opening_hours": "Mi 08:00-19:00"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    6.89573,
+                    50.84026
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Brühl-Vochem",
+                "location": "Thüringer Platz",
+                "opening_hours": "Fr 06:00-13:00"
+            },
+            "type": "Feature"
+        }
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Brühl - Wochenmärkte",
+            "url": "http://www.bruehl.de/wirtschaft/wirtschaftsfoerderung/wochenmaerkte.php"
+        },
+        "map_initialization": {
+            "coordinates": [
```

Comment:
![bruhl](https://cloud.githubusercontent.com/assets/144518/25557780/b761101c-2d18-11e7-905f-3c52712605a8.png)

The center point could be a bit lower for nice rendering on small screens.